File: java/Card.java
import java.sql.Date;

class Card extends Payment{
  Integer numbre;
  String name;
  Integer cvv;
  Date date;

  public Card(Integer id, Integer number, String name, Integer cvv, Date date){
    super(id);
    this.numbre = number;
    this.cvv = cvv;
    this.name = name;
    this.date = date;
  }
}

File: python/Card.py
from payment import Payment

class Card(Payment):
  name = str
  number = int
  cvv = int
  date = int

  def __init__(self, id, name, number, cvv, date):
    super().__init__(id)
    self.cvv = cvv
    self.date = date
    self.number = number
    self.name = name